overlays/crypto/blst.nix

```diff
@@ -2,6 +2,7 @@
 
 stdenv.mkDerivation rec {
   name = "blst-0.3.10";
```

Comment:
`name` is set by default to `${pname}-${version}" so you can just do `pname = "blst"` here.